src/main/java/tech/fastj/input/package-info.java

```diff
@@ -0,0 +1,5 @@
+package tech.fastj.input;
+
+/**
+ * Handles all input events in FastJ.
```

Comment:
To keep consistent with (most of) your other package info messages, remove "in FastJ".